### DISCUSSION TITLE
missing `after` `before` attr

### DISCUSSION THREAD

**Discussion by zyyv** [unocss/unocss#617] [Q&A] [answered]
```html
<div relative after="block content-none top-6 left-4 w-2 h-2 absolute bg-black">
  Unocss
</div>
```
This way of writing will cause the after attribute to be lost

<img width="548" alt="image" src="https://user-images.githubusercontent.com/42139754/154213136-76237c46-1332-4495-a8e9-01c565aae124.png">

**I don't know what is the reason for this ?**

I had to use another way instead
```html
<div relative after-block after-content-none after-top-6 after-left-4 after-w-2 after-h-2 after-absolute after-bg-black>
  Unocss
</div>
```
It's too complicated

ps: This case in [unocss.antfu.me](https://unocss.antfu.me/?html=DwEwlgbgBATgpgGwIYBdJyiuAPFBeAInhCgEYAOAB2wKgCM4AzAe3kLoWYGMBrKL5gDsswgLSChGFM0qiAbFARMUogCxQA7qIBMUABY6oSOgGdmCAK5Z6Ac1EckvAgD4AUFCgBVCVxMnXwAD04BBuASGwiKjoRoxYMPacvLHxogLCcGISghhIcXAJ0rIKealKcWopBaJauqXVBnX5CcZmltb1CXR2DrxuHt7cfgHBkP0eUEA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHCjjUrAFxwA2itV7azeqxjUAtOrGpaAGkRxmEMdB20oqZLTIBdA-6kAJRAA&options=N4IgzgLgTglgxhEAuaBXApgXyA) is right, but in project is error.

This is my minimal reproduction [demo](https://github.com/chris-zhu/unocss-pseudo-demo)

**Comment by antfu** [comment, marked as answer]
This is probably caused by Vue that removes the `after` attr in runtime for some reason. You can try:
```html
<div relative un-after="block content-none top-6 left-4 w-2 h-2 absolute bg-black">
  Unocss
</div>
```
